workshop.ipynb

```diff
@@ -267,9 +419,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Remote repository (25min)\n",
+    "#### 5.4 Command Breakdown\n",
+    "**git merge** - Merge is again of the most powerful commands in git. As the name indicates it alows us to merge or bring together two different versions of our files. It wil try to automatically put together and resolve differences in the files from the parameter branch and into our current branch. There are several options you an explore and even specific strategies to use when merging, but it's never perfect and conflict can arise often as you've experienced. After the command is succsessfully completed the current branch will be up to date with all the changes made in the parameter branch.\n",
+    "\n",
+    "**git status**  -Referenced [here](#4.1-Command-Breakdown)\n",
     "\n",
-    "Git works well at a local level but it wouldn't mean much if we couldn't work with other people. So we want to make our repository available online. In this workshop we will make use of GitHub an online free-to-use platform for hosting git repositories. Start by creating your accounts (if you don't have one already). Create an <strong>empty</strong> repository with a name of your choosing. After completing this steps lets add this repository as the remote for the local one we have been working on.\n"
+    "**git add** - Referenced [here](#4.2-Command-Breakdown)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 6. Remote repository (25min)\n",
+    "\n",
+    "Git works at a local level but it wouldn't mean much if we couldn't share out work with other people. So we want to make our repository available online. In this workshop we will make use of GitHub, an online, free-to-use platform for hosting git repositories. Start by creating an account (if you already have one that should be fine). <strong>Create an empty repository</strong> with a name of your choosing (one the home page use the green buttong on the left that sys new and do not fill any of the options that appear after the public, private selection). After completing this steps lets add this repository as the remote for the local one we have been working on.\n"
```

Comment:
"one" -> **on**
"buttong" -> **button**
"sys" -> **says**
"this" -> **these**